homeassistant/components/screenlogic/services.yaml

```diff
@@ -31,3 +31,20 @@ set_color_mode:
             - sunset
             - thumper
             - white
+start_super_chlorination:
+  target:
+    device:
+      integration: screenlogic
```

Comment:
The services don't seem to be device specific but config entry specific. Use the config entry selector instead.